packages/gatsby-remark-embed-snippet/src/utils/get-language.js

```diff
@@ -0,0 +1,30 @@
+"use strict"
+
+// Language defaults to extension.toLowerCase();
+// This map tracks languages that don't match their extension.
+const FILE_EXTENSION_TO_LANGUAGE_MAP = {
+  js: `jsx`,
+  md: `markup`,
+  sh: `bash`,
+  rb: `ruby`,
+  py: `python`,
+  ps1: `powershell`,
+  psm1: `powershell`,
+  bat: `batch`,
+  h: `c`,
+  tex: `latex`,
+}
+
+const getLanguage = file => {
+  if (!file.includes(`.`)) {
+    return `none`
+  }
+
+  const extension = file.split(`.`).pop()
+
+  return FILE_EXTENSION_TO_LANGUAGE_MAP.hasOwnProperty(extension)
+    ? FILE_EXTENSION_TO_LANGUAGE_MAP[extension]
+    : extension.toLowerCase()
+}
+
+module.exports = getLanguage
```

Comment:
let's avoid `module.exports` and use named export
```suggestion
exports.getLanguage = getLanguage
```

(and adjust the `get-lines.js` as well